Gemfile

```diff
@@ -32,105 +32,107 @@ gem 'jbuilder'
+gem "coveralls", require: false
+gem "kaminari"
 gem "selectize-rails"
-gem 'acts_as_list'
-gem 'acts_as_tree'
-gem 'activerecord-import',
-  git: 'https://github.com/zdennis/activerecord-import.git',
-  branch: 'master'
-gem 'thredded',
-  git: 'https://github.com/fosterfarrell9/thredded',
-  branch: 'master'
-gem 'kramdown-parser-gfm'
-gem 'thredded-markdown_katex'
-gem 'rails-i18n'
-gem 'kaminari-i18n'
-gem 'trix-rails', require: 'trix'
-gem 'xkcd'
-gem 'sunspot_rails'
-gem 'sunspot_solr'
-gem 'progress_bar'
-gem 'barby'
-gem 'rqrcode'
-gem 'sidekiq'
-gem 'faraday'
-gem 'globalize'
-gem 'globalize-accessors'
-gem 'commontator',
-  git: 'https://github.com/fosterfarrell9/commontator',
-  branch: 'master'
-gem 'acts_as_votable'
-gem 'sprockets-rails',
-  git: 'https://github.com/rails/sprockets-rails',
-  branch: 'master'
-gem 'premailer-rails'
-gem 'select2-rails'
-gem 'clipboard-rails'
-gem 'rubyzip'
+gem "acts_as_list"
+gem "acts_as_tree"
+gem "activerecord-import",
+  git: "https://github.com/zdennis/activerecord-import.git",
+  branch: "master"
+gem "thredded",
+  git: "https://github.com/fosterfarrell9/thredded",
+  branch: "master"
+gem "kramdown-parser-gfm"
+gem "thredded-markdown_katex"
+gem "rails-i18n"
+gem "kaminari-i18n"
+gem "trix-rails", require: "trix"
+gem "xkcd"
+gem "sunspot_rails"
+gem "sunspot_solr"
+gem "progress_bar"
+gem "barby"
+gem "rqrcode"
+gem "sidekiq"
+gem "faraday"
+gem "globalize"
+gem "globalize-accessors"
+gem "commontator",
+  git: "https://github.com/fosterfarrell9/commontator",
+  branch: "master"
+gem "acts_as_votable"
+gem "sprockets-rails",
+  git: "https://github.com/rails/sprockets-rails",
+  branch: "master"
+gem "premailer-rails"
+gem "select2-rails"
+gem "clipboard-rails"
+gem "rubyzip"
 
 group :development, :docker_development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'rspec-rails'
-  gem 'factory_bot_rails'
+  gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "rspec-rails"
+  gem "factory_bot_rails"
 end
 
 group :development, :docker_development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
-  gem 'listen', '>= 3.0.5', '< 3.2'
-  gem 'rails-erd'
+  gem "web-console", ">= 3.3.0"
+  gem "listen", ">= 3.0.5", "< 3.2"
+  gem "rails-erd"
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'rubocop', '~> 0.54.0', require: false
-  gem 'pgreset'
-  gem 'marcel'
-#  gem 'bullet'
+  gem "spring"
+  gem "spring-watcher-listen", "~> 2.0.0"
+  gem "rubocop", "~> 0.93", require: false
+  gem "rubocop-packaging", require: false
+  gem "rubocop-performance", require: false
+  gem "rubocop-rails", require: false
+  gem 'erb_lint', require: false
```

Comment:
If the quote setting in rubocop remains, then this should be a double quote. Otherwise, all the other quotes should be single quotes.